exoservice-javascript/bin/setup

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env bash
 
 yarn install
+./node_modules/.bin/build
+webpack
```

Comment:
I thought webpack contains a livescript compiler. So there shouldn't be a separate transpilation step via `node_modules/bin/.build` here.